packages/engine/Source/Scene/Fog.js

```diff
@@ -7,6 +7,9 @@ import SceneMode from "./SceneMode.js";
  * Blends the atmosphere to geometry far from the camera for horizon views. Allows for additional
  * performance improvements by rendering less geometry and dispatching less terrain requests.
  *
+ * View the {@link http://localhost:8080/Apps/Sandcastle/index.html?src=development%2FFog.html|Fog Sandcastle}
```

Comment:
In other places, we appear to use `@demo` tag. For example

```
 @demo {@link https://sandcastle.cesium.com/index.html?src=Box.html|Cesium Sandcastle Box Demo}
```

Also, like that example, please make sure to link to the deployed Sandcastle link, since user may be viewing docs online.